docs/setup/1-tezos-client.md

```diff
@@ -16,10 +16,13 @@ interface to Tezos.
 With [Homebrew](https://brew.sh):
 
 ```shell
-$ brew tap tqtezos/homebrew-tq https://github.com/tqtezos/homebrew-tq.git
-$ brew install tezos
+$ brew tap serokell/tezos-packaging https://github.com/serokell/tezos-packaging.git
+$ brew install tezos-client
 ```
 
+`tezos-packaging` provides prebuilt brew bottles for some of the macOS versions
+(Mojave and Catalina at the moment).
```

Comment:
Small suggestion:
```suggestion
`tezos-packaging` also provides prebuilt brew bottles for some macOS versions
(Mojave and Catalina at the moment).
```
Also, perhaps it would make more sense to not point out the versions supported now (as this doc will need to be updated), but instead where to find them, linking at the releases?

Reply:
I agree. Even linking to the release is probably not so interesting - brew install should work for vast majority of users, even if there's no bottle for a particular OS version - that will trigger compilation from source and will take longer, but ultimately should result in user having working software installed all the same